src/mlpack/tests/activation_functions_test.cpp

```diff
@@ -655,4 +706,20 @@ BOOST_AUTO_TEST_CASE(MishFunctionTest)
   CheckDerivativeCorrect<MishFunction>(desiredActivations,
                                         desiredDerivatives);
 }
-BOOST_AUTO_TEST_SUITE_END();
+
+/**
+ * Basic test of the CELU function.
+ */
+BOOST_AUTO_TEST_CASE(CELUFunctionTest)
+{
+  const arma::colvec desiredActivations("0.049787068 3.2 4.5 \
```

Comment:
I think you'll need to change the desiredActivations values, since you would be changing your initial function from 

`return (x >= 0) ? x : alpha * (std::exp((x / alpha) - 1));`

to

`return (x >= 0) ? x : alpha * (std::exp(x / alpha) - 1);`